MeannanApps/models.py

```diff
@@ -1,3 +1,78 @@
 from django.db import models
 
 # Create your models here.
+# creating model for Products
+
+class Products(models.Model):
+    product_name = models.CharField (max_length=50, blank=True, null=True)
+    product_price = models.FloatField()
+    product_image = models.ImageField(upload_to="products/")
+    description = models.TextField()
+
+    class Meta:
+        verbose_name = "Product"
+    verbose_name_plural = "Products"
+
+    def __str__(self) -> str:
+        return f"{self.product_name}, {self.product_price}"
+    
+class BillingAddress(models.Model):
+        name=models.CharField(max_length=200,blank=True,null=True)
+        emailaddress=models.EmailField( blank=True,null=True)
+        address=models.CharField(max_length=200,blank=True,null=True)
+        phone=models.IntegerField()
+        
+
+        class Meta:
+            verbose_name= "Billing Address"
+            verbose_name_plural= "Billing Addresses"
+
+        def __str__(self) -> str:
+                return f"{self.name }"   
+
+
+class ContactUs(models.Model):
+    name=models.CharField(max_length=200, blank=True,null=True)
```

Comment:
name can be just 50, this show the allocation of database to that field, we need to use the db space well